text/0032-access_command.md

```diff
@@ -0,0 +1,215 @@
+- Feature Name: access_command
+- Start Date: 2020-05-02
+- RFC PR: [#54](https://github.com/qri-io/rfcs/pulls/54)
+- Issue: <!-- (leave this empty) -->
+
+# Summary
+[summary]: #summary
+
+Decouple dataset visibility from data pushing, moving `publish` & `--unpublish` into a new subcommand: `access`. Position `access` for future commands.
```

Comment:
...for future access-control commands.
or permissions, rights, something.

Something to clarify that it will be for more than visibility.